mountpoint-s3/src/fuse.rs

```diff
@@ -225,7 +225,7 @@ where
         match block_on(self.fs.readdirplus(parent, fh, offset, replier).in_current_span()) {
             Ok(_) => {
                 reply.ok();
-                metrics::counter!("fuse.readdirplus.entries").increment(count as u64);
+                metrics::histogram!("fuse.readdirplus.entries").record(count as f64);
```

Comment:
We should make the same change for the `readdir` operation, as well as `readdirplus`.